runtime/doc/eval.txt

```diff
@@ -5741,29 +5741,33 @@ mode([expr])	Return a string that indicates the current mode.
 		a non-empty String (|non-zero-arg|), then the full mode is
 		returned, otherwise only the first letter is returned.
 
-			n	Normal
-			no	Operator-pending
-			v	Visual by character
-			V	Visual by line
-			CTRL-V	Visual blockwise
-			s	Select by character
-			S	Select by line
-			CTRL-S	Select blockwise
-			i	Insert
-			ic	Insert mode completion |compl-generic|
-			ix	Insert mode |i_CTRL-X| completion
-			R	Replace |R|
-			Rc	Replace mode completion |compl-generic|
-			Rv	Virtual Replace |gR|
-			Rx	Replace mode |i_CTRL-X| completion
-			c	Command-line editing
-			cv	Vim Ex mode |gQ|
-			ce	Normal Ex mode |Q|
-			r	Hit-enter prompt
-			rm	The -- more -- prompt
-			r?	A |:confirm| query of some sort
-			!	Shell or external command is executing
-			t	Terminal mode: keys go to the job
+			n	    Normal
+			no	    Operator-pending
+	         	nov	    Operator-pending (forced characterwise |o_v|)
+			noV	    Operator-pending (forced linewise |o_V|)
+			noCTRL-V    Operator-pending (forced blockwise |o_CTRL-V|);
+		   	    CTRL-V is one character
+			v	    Visual by character
+			V	    Visual by line
+			CTRL-V	    Visual blockwise
+			s	    Select by character
+			S	    Select by line
+			CTRL-S	    Select blockwise
+			i	    Insert
+			ic	    Insert mode completion |compl-generic|
+			ix	    Insert mode |i_CTRL-X| completion
+			R	    Replace |R|
+			Rc	    Replace mode completion |compl-generic|
+			Rv	    Virtual Replace |gR|
+			Rx	    Replace mode |i_CTRL-X| completion
+			c	    Command-line editing
+			cv	    Vim Ex mode |gQ|
+			ce	    Normal Ex mode |Q|
+			r	    Hit-enter prompt
+			rm	    The -- more -- prompt
+			r?	    A |:confirm| query of some sort
+			!	    Shell or external command is executing
+			t	    Terminal mode: keys go to the job
```

Comment:
`<tab>` in help files should be preserved, otherwise it causes unnecessary merge conflicts with Vim's help files.